recipes/aws-c-compression/all/test_v1_package/CMakeLists.txt

```diff
@@ -1,10 +1,8 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package LANGUAGES C)
+project(test_package)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
 
-find_package(aws-c-compression REQUIRED CONFIG)
-
-add_executable(${PROJECT_NAME} ../test_package/test_package.c)
-target_link_libraries(${PROJECT_NAME} PRIVATE AWS::aws-c-compression)
```

Comment:
Why removing this find_package()? I mean, if we are still generating the alias targets to maintain compatibility, shouldn't we still keep this?

Reply:
I don't understand, it's a common pattern we have in all modern recipes. It avoids code duplication and mistakes by ensuring a common CMakeLists in test package & test_v1_package, so find_package is not removed at all, but delegated to CMakeLists of test_package.

Reply:
oh sorry, I didn't read it well. It makes sense